### PR TITLE
Updates after integration with AFR on Cypress PSoC64 board

### PR DESCRIPTION
**Pull request by CharleyChu**
AFR IDT test pass

AFR MQTT Greengrass demos pass

### CODE REVIEW

File: iot_pkcs11_psa.c [hunk 1]
```diff
@@ -52,7 +52,7 @@
 #include "mbedtls/pk_internal.h"
 
 #define PKCS11_PRINT( X )            vLoggingPrintf X
-#define PKCS11_WARNING_PRINT( X )    vLoggingPrintf  X
+#define PKCS11_WARNING_PRINT( X )    /* vLoggingPrintf  X */
```
Comment (galak) [inline iot_pkcs11_psa.c:55]: Is this commented out on purpose or just for debugging?

Comment (CharleyChu) [reply iot_pkcs11_psa.c:55]: it generates too many messages, as a result, AFR IDT fails to parse the test result.

Comment (galak) [reply iot_pkcs11_psa.c:55]: Can anything be done about why the warnings are being generated instead of suppressing them?

Comment (galak) [reply iot_pkcs11_psa.c:55]: @CharleyChu any update?

Comment (CharleyChu) [reply iot_pkcs11_psa.c:55]: sorry, I thought you were asking Sherry. This is the warning message that generates a lot during test.

PKCS11_WARNING_PRINT( ( "WARNING: Search parameters other than label are ignored.\r\n" ) );

Are you OK to remove it, or downgrade it by adding "PKCS11_INFO_PRINT()"?

Comment (galak) [reply iot_pkcs11_psa.c:55]: I guess why or what's the call path that is causing that warning to get triggered?

Comment (Sherryzhang2) [reply iot_pkcs11_psa.c:55]: As commented in https://github.com/Linaro/freertos-pkcs11-psa/blob/master/iot_pkcs11_psa.c#L2101, this port only supports looking up objects by CKA_LABEL and all other search attributes are ignored. This keeps consistent with the mbedtls based implementation https://github.com/aws/amazon-freertos/blob/master/libraries/abstractions/pkcs11/mbedtls/iot_pkcs11_mbedtls.c#L2994

Comment (CharleyChu) [reply iot_pkcs11_psa.c:55]: the caller is xFindObjectWithLabelAndClass() in freertos_plus PKCS11 layer. its search template has both CKA_LABEL and CKA_CLASS https://github.com/aws/amazon-freertos/blob/master/libraries/freertos_plus/standard/pkcs11/src/iot_pkcs11.c#L300. but here we only support CKA_LABEL, So, the warning message will be print out for every call made from PKCS11 layer.

**Pull request merged**